subprojects/shipkit/src/main/groovy/org/shipkit/internal/notes/format/BadgeFormatter.java

```diff
@@ -17,11 +17,8 @@ public String getRepositoryBadge(String version, String publicationRepository, S
     }
 
     private String gradlePluginPortalBadge(String version, String publicationRepository, String pluginName) {
```

Comment:
Now `pluginName` is unused. So you can remove all usages and mark: 
- `get/setPublicationPluginName` in `AbstractReleaseNotesTask`
- `get/setPublicationPluginName` in `ReleaseNotes`
as deprecated.
I would remove it in the next MAJOR version.
Don't forget to bump the minor version of Shipkit to follow SemVer!
I'm working on https://github.com/mockito/shipkit/issues/233 (it should be ready to review in 2 weeks), so we can merge this PR + my together (to bump minor only once and add new value for users).

Reply:
Ok, now I see `ReleaseNotes` are fixed in https://github.com/mockito/shipkit/pull/785 but still more code should be removed in this PR.

Reply:
thank you, I will revisit the code and will remove not needed code.